src/api/getLink/__test__/getLink.spec.js

```diff
@@ -42,54 +45,53 @@ describe('getLink', () => {
             },
           ],
           payment: {
-            chainId: 80001,
-            tokenAddress: '0x9aa7fEc87CA69695Dd1f879567CcF49F3ba417E2',
+            chainId: Object.keys(config.contractInfo)[0],
+            tokenAddress: config.contractInfo[Object.keys(config.contractInfo)[0]].currency.USDT,
           },
           duration: 4353545453,
           userAddress: wallet.address,
         });
-      const nonce = 0;
+      const nonce = 0
 
-      const message = sha256(toUtf8Bytes(quoteId + nonce.toString()))
+      const message = utils.sha256(utils.toUtf8Bytes(requestQuotaResponse.body.quoteId + nonce.toString()))
       // Sign the original message directly
-      const signature = await signer.signMessage(message)
+      const signature = await wallet.signMessage(message)
 
       let response = await request(app).post(
-        `/getLink?quoteId=${
-          requestQuotaResponse.body.quoteId
+        `/getLink?quoteId=${requestQuotaResponse.body.quoteId
         }&nonce=${nonce}&signature=${signature}`
       );
       expect(typeof response.body).toBe('object');
-      expect(JSON.stringify(response.body)).toMatch(/Invalid Signature/i);
+      expect(JSON.stringify(response.body)).toMatch(/Invalid/i);
       expect(response.statusCode).toBe(400);
     });
 
     test('getLink for files', async () => {
+      const size = 100000
       let requestQuotaResponse = await request(app)
         .post(`/getQuote`)
         .send({
           type: 'filecoin',
           files: [
             {
-              length: 0,
+              length: size,
             },
           ],
           payment: {
-            chainId: 80001,
-            tokenAddress: '0x9aa7fEc87CA69695Dd1f879567CcF49F3ba417E2',
+            chainId: Object.keys(config.contractInfo)[0],
+            tokenAddress: config.contractInfo[Object.keys(config.contractInfo)[0]].currency.USDT,
           },
           duration: 4353545453,
           userAddress: wallet.address,
         });
-      const nonce = Date.now();
+      const nonce = 101 + await getLastKnowNonce(wallet.address.toLowerCase())
 
-      const message = sha256(toUtf8Bytes(quoteId + nonce.toString()))
+      const message = utils.sha256(utils.toUtf8Bytes(requestQuotaResponse.body.quoteId + nonce.toString()))
       // Sign the original message directly
-      const signature = await signer.signMessage(message)
+      const signature = await wallet.signMessage(message)
 
       let response = await request(app).post(
-        `/getLink?quoteId=${
-          requestQuotaResponse.body.quoteId
+        `/getLink?quoteId=${requestQuotaResponse.body.quoteId
```

Comment:
The same here needs a new line.

Reply:
@mariacarmina, we have a prettier configuration in the project to keep the format consistent 

We increased the printWidth to accommodate this.
```
  "printWidth": 100
```